smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -516,11 +630,7 @@ def doDeviceSync(){
 	log.trace "Doing Hue Device Sync!"
 	convertBulbListToMap()
 	poll()
-    try {
-		subscribe(location, null, locationHandler, [filterEvents:false])
-    } catch (all) {
```

Comment:
Not sure I did that but I see no reason for that unless it was added in response to @larsfinander investigating exceptions for Hue.